HW2_Slepov/calculator.py

```diff
@@ -0,0 +1,35 @@
+def multiplication(b):
+    a1 = float(b[0])
+    a2 = float(b[2])
+    return a1*a2
```

Comment:
```suggestion
    return a1 * a2
```

По PEP8 тут надо использовать пробелы